flow-typed/npm/winston_v3.x.x.js

```diff
@@ -74,7 +74,7 @@ declare type $winstonLoggerConfig<T: $winstonLevels> = {
   format?: $winstonFormat,
   level?: $Keys<T>,
   levels?: T,
-  transports?: Array<$winstonTransport>,
+  transports?: $winstonTransport | Array<$winstonTransport>,
```

Comment:
Are we allowed to do this?  Does winston actually allow a single transport object to be in `transports`?  I'm not sure why we would feel the need to do that.

Reply:
Yes, it does. I just had not updated the flow typing to match.

Reply:
Until now...